src/ACESVGjsPanel.tsx

```diff
@@ -1,29 +1,38 @@
 import React, { PureComponent } from 'react';
 import { PanelProps } from '@grafana/data';
-import { ACESVGOptions, SVGIDMapping } from 'types';
+import { ACESVGOptions, SVGIDMapping } from './types';
 import { css } from 'emotion';
 import { stylesFactory } from '@grafana/ui';
-import { SVG, Element as SVGElement, Dom as SVGDom, extend as SVGExtend, Runner as SVGRunner } from '@svgdotjs/svg.js';
+import {
+  SVG,
+  Element as SVGElement,
+  Dom as SVGDom,
+  extend as SVGExtend,
+  Runner as SVGRunner, // Use SVG prefix
+} from '@svgdotjs/svg.js';
 
 interface MappedElements {
   [key: string]: SVGElement | SVGDom;
 }
+
 interface Props extends PanelProps<ACESVGOptions> {}
+
 interface PanelState {
   addAllIDs: boolean;
   svgNode: SVGElement | SVGDom | null;
   svgSource: string | null;
   mappedElements: MappedElements | null;
-  svgMappings: SVGIDMapping[];
```

Comment:
removed it from here -- not sure why you would want a different copy than the props

Reply:
Hmmm I need to rebuild mappedElements when it's updated, I was using the state to detect those changes. If there's a better way to do that....